libcaja-private/caja-file.c

```diff
@@ -2534,7 +2534,7 @@ update_info_internal (CajaFile *file,
 				(file->details->directory, file);
 
 			g_clear_pointer (&file->details->name, g_ref_string_release);
-			if (eel_strcmp (file->details->display_name, name) == 0) {
+			if (g_strcmp0 (file->details->display_name, name) == 0) {
```

Comment:
Why this change?  I'm not against it as `g_strcmp0()` is more standard, but both work on `NULL`, yet they have slightly different behavior: `eel_strcmp()` considers `NULL`s equal to `""` while `g_strcmp0()` sorts `NULL` before anything else.

If the goal is not to use a `NULL` `file->details->display_name`, maybe explicit that?

Reply:
I was wondering if this would be a problem, because there are a bunch of other usages of `eel_strcmp()` that I didn't touch and it felt weird to only change that one spot. Nautilus already changed across the board to `g_strcmp0()`, so it wasn't affected by this part of the issue. You're absolutely right, the goal is to ensure we don't end up with a `NULL` `display_name`, because that results in an assertion failure later. I will change to explicitly check for NULL instead, and leave `eel_strcmp()` alone. Thanks for the review!